front/src/routes/integration/all/zwave/setup-page/Node.jsx

```diff
@@ -22,9 +22,7 @@ const createGithubUrl = node => {
   return `${GITHUB_BASE_URL}?title=${title}&body=${body}`;
 };
 
-const displayRawNode = node => () => {
-  console.log(node);
-};
+const displayRawNode = node => () => {};
```

Comment:
Why ? This was done on purpose so that the user can see his full raw node in the console to help us debug!

Opening the full raw Z-Wave node in GitHub doesn't work because it's too big to fit it an URL

Reply:
OK sorry. I thought the URL build was already containing the needed data.
In this case, @VonOx you must revert to keep the console.log and add this line above
`// eslint-disable-next-line no-console`

Reply:
No worries! I know it's really not ideal, we need to find another solution after that